.claude/agents/review-performance.md

```
@@ -0,0 +1,54 @@
+---
+name: review-performance
+description: Performance-focused code review. Checks for unnecessary allocations, clone-heavy patterns, missing caches, hot path inefficiencies, and O(n²) algorithms.
+model: sonnet
+color: yellow
+---
+
+Use the local todo or task tool for the current slice. Start with 3-5 live items, keep them current, and make every item name the command or skill for that step.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)